test/fuzz/Liquidator/LiquidateAccount.fuzz.t.sol

```diff
@@ -55,7 +55,7 @@ contract LiquidateAccount_Liquidator_Fuzz_Test is Liquidator_Fuzz_Test {
         assertEq(isAuctionActive, true);
 
         uint256 startDebt = liquidator.getAuctionStartPrice(address(proxyAccount));
-        uint256 loan = uint256(amountLoaned + 1) * 150 / 100;
+        uint256 loan = uint256(amountLoaned + 1) * 150;
```

Comment:
What happens here?

Reply:
I removed both occurence of that check. It's to validate that startPrice is 150% of startDebt. But I don't think it makes sense here. And the `getAuctionStartPrice()` from the extension also. Testing should be covered in `calculateBidPrice()`